include/pgduckdb/pgduckdb_duckdb.hpp

```diff
@@ -41,7 +48,8 @@ class DuckDBManager {
 
 	int secret_table_num_rows;
 	int secret_table_current_seq;
-	duckdb::unique_ptr<duckdb::DuckDB> database;
+	duckdb::DuckDB *database;
```

Comment:
Can we add a FIXME to restore the smart pointer here?

Reply:
Done